init_dev_container.sh

```diff
@@ -4,7 +4,9 @@
 psql "postgres://dev:dev@localhost:5432/" -c "CREATE DATABASE dust_api;";
 psql "postgres://dev:dev@localhost:5432/" -c "CREATE DATABASE dust_databases_store;";
 psql "postgres://dev:dev@localhost:5432/" -c "CREATE DATABASE dust_front;";
+psql "postgres://dev:dev@localhost:5432/" -c "CREATE DATABASE dust_front_migration;";
```

Comment:
How are these going to be initialized? How am I guaranteed without extra effort that they are in sync with my main db to properly apply the diff?

Reply:
The create_db_migration_script run `init_db.sh` which sync everything (so you are guaranteed to be up-to-date) and it does a diff with your branch.

Right now, this "sync" is applied to your "real" db as a (not great) side effect.